### ISSUE TITLE
Documentation changes: nginx websocket config

### ISSUE THREAD

**Issue by mworrell** [zotonic/zotonic#2140]
From @karlsson:

```
        # websocket
        location /mqtt-transport {
                 proxy_pass       http://127.0.0.1:8000/mqtt-transport;
                 proxy_set_header Host      $host;
                 proxy_set_header X-Real-IP $remote_addr;
                 proxy_set_header X-Forwarded-For  $proxy_add_x_forwarded_for;
                 proxy_set_header X-Forwarded-Proto $scheme;
                 proxy_http_version 1.1;
                 proxy_set_header Upgrade $http_upgrade;
                 proxy_set_header Connection "upgrade";
                 proxy_set_header Sec-Websocket-Extensions $http_sec_websocket_extensions;
                 proxy_set_header Sec-Websocket-Key $http_sec_websocket_key;
                 proxy_set_header Sec-Websocket-Protocol $http_sec_websocket_protocol;
                 proxy_set_header Sec-Websocket-Version $http_sec_websocket_version;
        }
```

**Comment by karlsson** [comment]
@mworrell 
OK, I can pick this one. When I try to generate docs (make docs) to check the generated html before pushing my pull request I get errors for missing directories or files though:
`export ZOTONIC_SRC=../ MODULES=../apps/zotonic_mod* ; for f in `find ref -name .generate-erlang`; do $f || exit 1; done
export APPS=../apps CORE=../apps/zotonic_core MODULES=../apps/zotonic_mod* ; for f in `find ref -name .generate`; do $f || exit 1; done
ref/templates/.generate: rad 18: ref/templates/meta-item_list-schema_org/types.rst: Filen eller katalogen finns inte
New doc: ref/templates/template_item_list-schema_org/types.rst`

Now the updates are not big so I can push anyway and have you review the pull request?

**Comment by mworrell** [comment]
Closed in #2143
